src/steps/build.py

```diff
@@ -286,25 +287,38 @@ def runp(cmd):
     return runp('bundle install')
 
 
+def update_jekyll_config(federalist_config={}, custom_config=''):
+    logger = get_logger('build-jekyll')
+
+    JEKYLL_CONF_YML_PATH = CLONE_DIR_PATH / JEKYLL_CONFIG_YML
+
+    config_yml = {}
+    with JEKYLL_CONF_YML_PATH.open('r') as jekyll_conf_file:
+        config_yml = yaml.safe_load(jekyll_conf_file)
+
+    if custom_config:
+        try:
+            custom_config = json.loads(custom_config)
+        except json.JSONDecodeError:
+            logger.warning('Could not load/parse custom yaml config.')
```

Comment:
Do we want to warn and ignore this? Should the build fail? Should we check at the very beginning to fail it then??

@amirbey ^

Reply:
@davemcorwin - check at the very beginning where?

Reply:
We could try to parse and dump it before the build actually starts to make sure it won't fail. I think this is unlikely since the webapp verifies the yaml is valid, but something could go wrong in converting back and forth I suppose.

Reply:
The bigger question for me is should the build fail if the user provided config can't be used?

Reply:
i think the build failure may be best...  incase not having the environment variable yields an undesired versiona user's site (in production)